torch/onnx/utils.py

```diff
@@ -947,7 +947,7 @@ def _newNode(g, opname, outputs, *args, **kwargs):
         aten = False
         ns_opname = opname
     else:
-        aten = kwargs.pop("aten", False)
```

Comment:
Is there a case where first condition validates to False, while second is True?

Reply:
Not after unifying onnx::aten and aten::aten. Good point, removing it